src/log/logger_config.cpp

```diff
@@ -35,10 +35,7 @@ namespace fc {
          log = log_config::get().logger_map[name];
       } else {
          // no entry for logger, so setup with log_config's registered appenders
-         log = logger(name);
-         for( const auto& a : log_config::get().appender_map ) {
-            log.add_appender( a.second );
-         }
+         log = log_config::get().logger_map[ DEFAULT_LOGGER ];
```

Comment:
What if it does not exist because a user provided a custom logging.json that excluded a logger named `default`?

Reply:
I'm open to suggestions. Could go with what I had before in that case. Or grab first one from the map. I think I like what I had before in that case.

Reply:
Went with default constructed and existing appenders if no DEFAULT_LOGGER found.

Reply:
After discussion, went with not setting up any logger if no default provided.